airflow/exceptions.py

```diff
@@ -298,3 +298,7 @@ def __repr__(self) -> str:
 
 class TaskDeferralError(AirflowException):
     """Raised when a task failed during deferral for some reason."""
+
+
+class DAGDisallowManual(AirflowException):
```

Comment:
```suggestion
class ManualDAGRunDisallowed(AirflowException):
```

seems to read better forme.